src/core/ext/filters/client_channel/lb_policy/round_robin/round_robin.c

```diff
@@ -165,20 +165,20 @@ static size_t get_next_ready_subchannel_index_locked(
   if (GRPC_TRACER_ON(grpc_lb_round_robin_trace)) {
     gpr_log(GPR_INFO,
             "[RR: %p] getting next ready subchannel, "
-            "last_ready_subchannel_index=%zu",
-            p, p->last_ready_subchannel_index);
+            "last_ready_subchannel_index=%d",
+            p, (int)p->last_ready_subchannel_index);
```

Comment:
ahh, it looks like `%llu` isn't supported either. (it failed to compile, found some confirmation in https://stackoverflow.com/questions/13590735/printf-long-long-int-in-c-with-gcc).

The `%lu` format specified appears to be ok though...